src/stories/index.js

```diff
@@ -53,6 +61,18 @@ storiesOf('StackedArea', module)
         />
     ));
 
+storiesOf('Donut', module)
```

Comment:
Now this stuff goes into a markdown file, check out https://github.com/eventbrite/britecharts-react/blob/master/src/charts/stackedArea/Readme.md

Reply:
I'll have a look, thank you for the feedback!